src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,67 @@
+package duke.command;
+
+import duke.Deadline;
+import duke.DukeException;
+import duke.Event;
+import duke.Task;
+import duke.TaskList;
+import duke.TaskType;
+import duke.ToDo;
+import duke.Ui;
+
+public class AddCommand implements Command {
+    private String[] params;
+
+    public AddCommand(String[] params) {
+        this.params = params;
+    }
+
+    public TaskType getTaskType(String tasktypeInput) {
+        return tasktypeInput.equals("event") ? TaskType.Event
+                : tasktypeInput.equals("deadline") ? TaskType.Deadline
+                : tasktypeInput.equals("todo") ? TaskType.ToDo
+                : TaskType.Invalid;
+    }
+    @Override
+    public void execute(TaskList taskList, Ui ui) throws DukeException {
+        TaskType type = getTaskType(params[0]);
+        if (type.equals(TaskType.Invalid)) {
+            throw new DukeException("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+        }
+        if (params.length == 1 || params[1].isEmpty()) {
+            throw new DukeException("☹ OOPS!!! The description of a task cannot be empty.");
+        }
+        Task t;
+        String taskInfo = params[1];
+        switch (type) {
+            case Event:
+                String[] eventInfo = taskInfo.split(" /at ");
```

Comment:
Should the indentation for `case` be removed instead?

Reply:
Fixed! 🙂